logtail/logtail.go

```diff
@@ -461,12 +461,25 @@ func (l *Logger) upload(ctx context.Context, body []byte, origlen int) (uploaded
 	return true, nil
 }
 
-// Flush uploads all logs to the server.
-// It blocks until complete or there is an unrecoverable error.
+// Flush uploads all logs to the server. It blocks until complete or there is an
+// unrecoverable error.
+//
+// TODO(bradfitz): this apparently just returns nil. Not sure the history of it
```

Comment:
https://github.com/tailscale/corp/commit/9c2ec3565bb94f1d78de18988eb6b6b793209ed6 appears to be when it became a no-op